lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -497,7 +493,12 @@ export class StandardController implements IController {
                     EventType.HANDLE_REDIRECT_END,
                     InteractionType.Redirect
                 );
-                rootMeasurement.end({ success: false });
+
+                if (rootMeasurement.event.errorCode) {
+                    rootMeasurement.end({ success: false });
+                } else {
+                    rootMeasurement.discard();
```

Comment:
Will this not discard all successful measurements as well? Or once a measurement has been ended it can no longer be discarded?

Can you also please add a comment describing the what & why we're doing here?

Reply:
Measurements are grouped by a correlation id. We are discarding a single measurement here including all sub-measurements.

Reply:
Added a comment.